lib/ui/views/action_info/action_info_viewmodel.dart

```diff
@@ -30,8 +31,8 @@ class ActionInfoViewModel extends FutureViewModel<Action> {
     try {
       await _causesService.completeAction(data!.id);
       setBusy(false);
-      // TODO Really?
-      _routerService.back();
+      _dialogService.showActionCompletedDialog();
+      notifySourceChanged();
```

Comment:
@JElgar Could you please double check if this is correct way to refresh the data? This is working fine, but maybe there's some other method from Stacked that I should use?

Reply:
I think the method we have been using is `notifyListeners` (which comes from stacked `BaseViewModel`)

e.g. https://github.com/now-u/now-u-app/blob/main/lib/ui/views/home/home_viewmodel.dart#L118

Reply:
I've tried it as a first thing, but unfortunately that wasn't updating the screen